vmdb/lib/miq_automation_engine/models/miq_ae_class_copy.rb

```diff
@@ -78,15 +78,14 @@ def add_fields
 
   def validate
     dest_class = MiqAeClass.find_by_fqname("#{@target_ns_fqname}/#{@target_name}")
-    if dest_class
-      dest_class.destroy if @overwrite
-      raise "Destination Class already exists #{dest_class.fqname}" unless @overwrite
-    end
+    return unless dest_class
+    dest_class.destroy if @overwrite
```

Comment:
@mkanoor I suspect rubocop is Just Wrong, but maybe this is a partial improvement?

```ruby
raise "Destination Class already exists #{dest_class.fqname}" unless @overwrite
dest_class.destroy
```

(btw, if we're not checking the result, shouldn't this be using `destroy!`?)